salt/docker_apps/files/kingfisher_process.yaml

```diff
@@ -3,10 +3,8 @@ version: "2.2"
 x-shared: &shared
   image: "ghcr.io/open-contracting/kingfisher-process:latest"
   network_mode: "host"
-  env_file:
-    - env
   volumes:
-    - /data:/data
+    - {{ pillar.kingfisher_collect.files_store }}:/data
```

Comment:
Why not `pillar.kingfisher_collect.env.FILES_STORE` to avoid creating another Pillar key?

Reply:
True, changed it now